tests/installation/logs_from_installation_system.pm

```diff
@@ -47,12 +47,14 @@ sub run {
         # set serial console for xen
         set_serial_console_on_vh('/mnt', '', 'xen') if (get_var('XEN')                      || check_var('HOST_HYPERVISOR', 'xen'));
         set_serial_console_on_vh('/mnt', '', 'kvm') if (check_var('HOST_HYPERVISOR', 'kvm') || check_var('SYSTEM_ROLE',     'kvm'));
+        set_dom0_mem('/mnt') if (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen'));
```

Comment:
Our tests fit the xen condition, but may not always work with 4G dom0 mem because some old SUTs on osd only has 4G mem. If this apply only to REGRESSION test, can you add this condition too ?

Reply:
```suggestion
        set_dom0_mem('/mnt') if (get_required_var('REGRESSION') && (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen')));
```
Is this OK?